src/workerd/api/sql-test.js

```diff
@@ -143,22 +143,14 @@ async function test(storage) {
   )
 
   // Prepared statement with whitespace
-  const preparedWithSpace = sql.prepare('SELECT 1; ')
-  const resultPreparedWithSpace = [...preparedWithSpace()]
+  const whitespace = [' ', '\t', '\n', '\r', '\v', '\f']
 
-  const preparedWithCr = sql.prepare('SELECT 1;\r')
-  const resultPreparedWithCr = [...preparedWithCr()]
+  for (const char of whitespace) {
+    const prepared = sql.prepare(`SELECT 1;${char}`);
+    const result = [...prepared()]
 
-  const preparedWithLf = sql.prepare('SELECT 1;\n')
-  const resultPreparedWithLf = [...preparedWithLf()]
-
-  const preparedWithCrlf = sql.prepare('SELECT 1;\r\n')
```

Comment:
I think this testcase is not present anymore. Am I missing something?

Reply:
Hi Vicky, good catch! I've added it back in.